src/components/Search/SelectedRevisions.tsx

```diff
@@ -21,8 +29,9 @@ function SelectedRevisions({
   isWarning,
   view,
 }: SelectedRevisionsProps) {
-
   const styles = SelectRevsStyles(mode);
+  const [revisions, setRevisions] = useState<RevisionsList[]>([]);
+  const [repositories, setRepositories] = useState<Repository['name'][]>([]);
```

Comment:
(Not for this PR since it was already like this) Is there a reason why we keep them separately? It looks like we can put them both in a single array, since they have to be in sync all the time.
For exasmple, we can create a new type called `RevisionWithRepo` that can hold both.

Reply:
True, I'll update in the next PR. Thanks.